api/apiclient_test.go

```diff
@@ -94,7 +95,7 @@ func (s *apiclientSuite) TestConnectWebsocketMultipleError(c *gc.C) {
 	info := s.APIInfo(c)
 	addr := listener.Addr().String()
 	info.Addrs = []string{addr, addr, addr}
-	_, _, err = api.ConnectWebsocket(info, api.DialOpts{})
+	_, _, err = api.DialAPI(info, api.DialOpts{})
 	c.Assert(err, gc.ErrorMatches, `unable to connect to API: websocket.Dial wss://.*/model/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}/api: .*`)
```

Comment:
Having just gone through fixing a bunch of tests because the error message is now prepended with something different, please modify this to only check the relevant portion of the error message.